_data/engine-cli/docker_container_create.yaml

```diff
@@ -63,9 +63,6 @@ options:
 - option: device
   default_value: '[]'
   description: Add a host device to the container
-- option: device-cgroup-rule
-  default_value: '[]'
-  description: Add a rule to the cgroup allowed devices list
```

Comment:
This feature was not in 17.03 yet, but is part of 17.04 (https://github.com/docker/docker/pull/22563). The old YAML files were probably generated from master so contained the incorrect information. That was one of the issues we discovered when reviewing the YAML generator PR (i.e., it always built the generator from 'master', not from the code present in the branch / current commit)